extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1000,6 +1006,10 @@ export class AmpA4A extends AMP.BaseElement {
 
   /** @override  */
   unlayoutCallback() {
+    if (!isExperimentOn(this.win, 'a4a-fie-unlayout-enabled') &&
```

Comment:
Correct me if I'm wrong, Keith, but based off Keith's reply to my question above, I think the approach here is that this new behavior will run when the experiment flag doesn't exist, and we'll only add the flag if we need to shut it off.